src/authentication/configuration.ts

```diff
@@ -15,12 +20,13 @@ export const HostHelper = class {
 				return vscode.Uri.parse(testEnv);
 			}
 
+			const yes = vscode.l10n.t('Yes');
 			const result = await vscode.window.showInformationMessage(
-				`The 'GITHUB_TEST_SERVER' environment variable is set to '${testEnv}'. Use this as the GitHub API endpoint?`,
+				vscode.l10n.t('The \'GITHUB_TEST_SERVER\' environment variable is set to \'{0}\'. Use this as the GitHub API endpoint?', testEnv),
```

Comment:
btw you can use backticks with `vscode.l10n.t(...)` doesn't have to use `''`. I just like using `''` by default and then either `""` or backticks when the string contains `'` but totally your call.

Reply:
Good to know, thank you!